src/preprocess/vhi.py

```diff
@@ -0,0 +1,173 @@
+"""
+
+- Add lat lon coordinates
+- add time coordinates
+- subset Kenya
+- merge into one time (~500MB)
+"""
+from pathlib import Path
+import pathlib
+import xarray as xr
+import multiprocessing
+from typing import List, Optional, Tuple, Union
+import pickle
+from functools import partial
+
+from xarray import Dataset
+
+from .base import (BasePreProcessor, get_kenya)
+from .preprocess_vhi import (
+    extract_timestamp,
+    create_lat_lon_vectors,
+    create_new_dataset,
+    create_filename,
+)
+from .preprocess_utils import select_bounding_box_xarray
+
+
+class VHIPreprocessor(BasePreProcessor):
+    """ Preprocesses the VHI data """
+
+    def __init__(self, data_folder: Path = Path('data')) -> None:
+        super().__init__(data_folder)
+
+        self.out_dir = self.interim_folder / "vhi_preprocessed"
+        if not self.out_dir.exists():
+            self.out_dir.mkdir()
+
+        self.vhi_interim = self.interim_folder / "vhi"
+        if not self.vhi_interim.exists():
+            self.vhi_interim.mkdir()
+
+    def get_vhi_filepaths(self) -> List[Path]:
+        return [f for f in (self.raw_folder / "vhi").glob('*/*.nc')]
+
+    @staticmethod
+    def preprocess_VHI_data(netcdf_filepath: str,
+                            output_dir: str,
+                            subset: str = 'kenya',) -> Path:
```

Comment:
Since right now, `subset` has to be 'kenya' to do anything, can we take it out of the function arguments? Or make it a boolean argument, e.g.
`subset_kenya: bool = True`